src/actionlist/action-list-item.scss

```diff
@@ -300,11 +335,10 @@
       color: var(--color-danger-fg);
     }
 
-    @media (hover: hover) and (pointer: fine) {
```

Comment:
Did you remove `and (pointer: fine)` because we weren't getting hover styles on touch devices with an optional mouse?

Reply:
@mperrotti no... I removed it because the `pointer` query shouldn't influence whether we display these `:hover` styles or not. Devices that have `pointer: coarse` which also support `hover: hover` should still display it. See the [combination table](https://drafts.csswg.org/mediaqueries/#mf-interaction) from W3C.

:v: